.storybook/main.ts

```diff
@@ -28,31 +28,33 @@ const config: StorybookConfig = {
     }
 
     // configure .svg files to be loaded with @svgr/webpack
-    config?.module?.rules?.push({
-      test: /\.svg$/,
-      use: ["@svgr/webpack"],
-    },
-    {
-      test: /\.scss$/,
-      use: [
-        "style-loader",
-        // Translates CSS into CommonJS
-        "css-loader",
-        {
-          // Compiles Sass to CSS
-          loader: "sass-loader",
-          options: {
-            data: `@import "./assets/scss/app.scss";`,
-            includePaths: [__dirname, "./assets/**/*"],
+    config?.module?.rules?.push(
+      {
+        test: /\.svg$/,
+        use: ["@svgr/webpack"],
+      },
+      {
+        test: /\.scss$/,
```

Comment:
what's up with the scss? Ideally we don't want to have both scss and styled components at the same time.

Reply:
I have a style override for Sandpack.